mne/channels/channels.py

```diff
@@ -744,3 +744,39 @@ def _ch_neighbor_connectivity(ch_names, neighbors):
 
     ch_connectivity = sparse.csr_matrix(ch_connectivity)
     return ch_connectivity
+
+
+def fix_mag_coil_types(info):
+    """Fix Elekta magnetometer coil types
+
+    Parameters
+    ----------
+    info : dict
+        The info dict to correct. Corrections are done in-place.
+
+    Notes
+    -----
+    This function changes magnetometer coil types 3022 (T1: SQ20483N) and
+    3023 (T2: SQ20483-A) to 3024 (T3: SQ20950N) in the channel definition
+    records in the info structure.
+
+    Neuromag Vectorview systems can contain magnetometers with two
+    different coil sizes (3022 and 3023 vs. 3024). The systems
+    incorporating coils of type 3024 were introduced last. At some sites
```

Comment:
From Samu/Jukka, T3 coil types are used on most Neuromag systems, so maybe add that detail. Something like " ... were introduced last and are used at the majority of MEG sites."